src/Pinoco/Validator.php

```diff
@@ -614,6 +614,7 @@ private function _testEmptyComplex($exists, $value)
         if ($value === "0" || $value === 0 || $value === false || $value === array()) {
             return false;
         }
+        $value = trim(mb_convert_kana($value, "s", 'UTF-8'));
```

Comment:
@itigoppo Text triming is not pure validity test. Instead these issues should be handled by `filter()` method, I think.

```php
$validator->check('foo')->filter('your-triming-rule')->is('not-empty');
```